### ISSUE TITLE
How to train  with low quality data?

### ISSUE THREAD

**Issue by wotulong** [kan-bayashi/ParallelWaveGAN#123]
Any suggestions of training the model with low qulity, such as audio recoding by iphone.I have try to train with such dataset, and the discriminator_loss decrease to almost zero(0.006) after 110K steps, which means it's hard to get a good result.

**Comment by kan-bayashi** [comment]
Could you provide tensorboard log? (Screenshot is enough)
How many utterances did you use for training?
Single speaker or Multi spekaer?
Did it contain serious background noise?
I've never tried but it is worthwhile to fine-tune using pretrained model.

**Comment by wotulong** [comment]
> Could you provide tensorboard log? (Screenshot is enough)
> How many utterances did you use for training?
> Single speaker or Multi spekaer?
> Did it contain serious background noise?
> I've never tried but it is worthwhile to fine-tune using pretrained model.

![111](https://user-images.githubusercontent.com/4609486/77381398-bff2ec80-6db8-11ea-89d9-e780bb33a458.jpg)
1:Training utterances:200K(about 130 hours);
2:Total speakers:600
3:It contain some background noise, not serious(ps:not as good as LibriSpeech)
Does the model very sensitive to noise? Like wavenet.

**Comment by kan-bayashi** [comment]
Thank you. Very large dataset!
I forgot to ask one more question.
Which configuration did you use? (`parallel_wavegan.v1.yaml` base?)
Actually, I have not yet trained with noisy data, but in my experiment, it is better to make the dataset clean as much as possible.
For example, trim the silence carefully (force alignment is the best), and normalize the volume of the waveform of each speaker.

**Comment by kan-bayashi** [comment]
Another point is that I successfully trained the model with environmental sounds, not human speech.
Therefore, I think it can be trained with noisy speech (but maybe the generated speech will also be noisy).

**Comment by wotulong** [comment]
> Thank you. Very large dataset!
> I forgot to ask one more question.
> Which configuration did you use? (`parallel_wavegan.v1.yaml` base?)
> Actually, I have not yet trained with noisy data, but in my experiment, it is better to make the dataset clean as much as possible.
> For example, trim the silence carefully (force alignment is the best), and normalize the volume of the waveform of each speaker.

Thank you for quickly reply.

1. - The config file I use is parallel_wavegan.v2.yaml.
2. - I hvae use the "WaveEdit" to trim begin and end silence before train I train the model and  reduce the noise by the software as well。
3. - The outputs of the model sound likr loss some details。Below is the plot of the model output  and ground truth data。

![compare](https://user-images.githubusercontent.com/4609486/77392877-e32c9480-6dd6-11ea-86e7-7c9e9a2de4a1.png)

**Comment by kan-bayashi** [comment]
`parallel_wavegan.v2.yaml` is deprecated. I could not find a stable setting for v2's discriminator, so I removed `v2` configs (See discussion in #27 #61).
Please try with `parallel_wavegan.v1.yaml`.

**Comment by wotulong** [comment]
> `parallel_wavegan.v2.yaml` is deprecated. I could not find a stable setting for v2's discriminator, so I removed `v2` configs (See discussion in #27 #61).
> Please try with `parallel_wavegan.v1.yaml`.

OK, Thanks, I will try with it. And I will reply after I finish new training.

**Comment by wotulong** [comment]
> `parallel_wavegan.v2.yaml` is deprecated. I could not find a stable setting for v2's discriminator, so I removed `v2` configs (See discussion in #27 #61).
> Please try with `parallel_wavegan.v1.yaml`.

"parallel_wavegan.v1.yaml" works. The loss curve looks normal now, but there are still some noise in 
 the output and the quality is not as good as the training data. Maybe the training data have too much noise.I'll do more experiments with the training data.

**Comment by maxbaluev** [comment]
@wotulong please keep n touch with your experiments based on the multi speaker model.

**Comment by wotulong** [comment]
> @wotulong please keep n touch with your experiments based on the multi speaker model.

The quality of the voice ouput is depend on training data, it sounds good when training data is clean. And multi speaker model is working well on the speaer that not in the traing data(I hvae try model with 600 speakers andl 60 speaker).
